### PR TITLE
IDE-2862 add default value for project name in Config source Page

### PR DESCRIPTION
**Pull request by lovettli**

### CODE REVIEW

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/upgrade/ICodeUpgradleConstants.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+package com.liferay.ide.project.core.upgrade;
+
+/**
+ * @author Lovett Li
+ */
+public interface ICodeUpgradleConstants
+{
+    String LAYOUT_UPGRADE_TO_LIFERAY_PLUGIN_SDK_7 = "Upgrade to liferay plugin sdk 7";
+    String PROJECT_NAME_PLUGIN_SDK_7 = "plugins.sdk-7.0";
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/upgrade/ICodeUpgradleConstants.java]: This shouldn't be "plugins.sdk-7.0" should it?

Comment (lovettli) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/upgrade/ICodeUpgradleConstants.java]: This name we just give a default project name when user open the upgrade view and select upgrade to liferay plugin sdk. We can change it to any value.

Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/upgrade/ICodeUpgradleConstants.java]: I think it should be "plugins-sdk" that is what the project is called by default in `blade init -u` command.

**Comment by gamerson** [comment]
See comments

**Pull request closed without merging**